### ISSUE TITLE
Missing test / example models for RelTemperatureSensor, HeatFlowSensor and ConditionalFixedHeatFlowSensor of HeatTransfer

### ISSUE THREAD

**Issue by beutlich** [modelica/ModelicaStandardLibrary#2681]
There are no test or example models for the relative sensors Modelica.Thermal.HeatTransfer.Sensors.RelTemperatureSensor, Modelica.Thermal.HeatTransfer.Sensors.HeatFlowSensor and Modelica.Thermal.HeatTransfer.Sensors.ConditionalFixedHeatFlowSensor in the MSL.

**Comment by AHaumer** [comment]
You're right. Sorry I don't have the time to invent meaningful test example right now.
